### DISCUSSION TITLE
Dynamically register flows

### DISCUSSION THREAD

**Discussion by norangai** [PrefectHQ/prefect#4444] [Q&A]
I am currently working on a ETL project where I need to pull ad insights from 600+ ad accounts. I tried to use mapped tasks but it can only run 2 tasks at a time(using executor = LocalDaskExecutor()). Therefore, I would like to dynamically register a flow for each account thinking that it might enable more accounts to be processed at the same time. However, I am not sure how to do it, or if it is a good approach. Any suggestions or advice would be highly appreciated!

**Comment by zanieb** [comment]
Hey @norangai -- you should be able to run thousands of tasks in parallel. Registering additional flows is certainly possible but is not considered good practice. You can set additional workers on your dask executor with `num_workers` e.g. 

```python
flow.executor = LocalDaskExecutor(num_workers=20)
```
